File: week_1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height;
    do
    {
        Height = get_int("Height: ");
    }
    while (Height <= 0 || Height > 8);

    for (int i = 0; i < Height; i++)
    {
        for (int j = 0; j < Height + (i + 3); j++)
        {
            if (j >= Height - (i + 1) && j <= Height - 1)
            {
                printf("#");
            }
            else if (j >= Height + 2 && j <= Height + (i + 2))
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}

File: week_6_Python/mario/mario.py
from cs50 import get_int
while True:
    Height = get_int("Height : ")
    if Height > 0 and Height < 9:
        break

for i in range(Height):
    for j in range(Height + i + 3):
        if j >= Height - (i + 1) and j <= Height - 1:
            print("#", end='')
        elif j >= Height + 2 and j <= Height + (i + 2):
            print("#", end='')
        else:
            print(" ", end='')
    print("\n", end='')